docs/new_problem.md

```diff
@@ -240,16 +240,38 @@ All hyperparamters inherit from `_default_hparams()` in `problem.py.` If you wou
 from tensor2tensor.models import transformer
 
 @registry.register_hparams
-def word2def_hparams(self):
+def word2def_hparams():
     hparams = transformer.transformer_base_single_gpu()  # Or whatever you'd like to build off.
     hparams.batch_size = 1024
     return hparams
 ```
 
+# Test the data generation
+If you are using a local checkout of tensor2tensor, you can install the local version in developer mode with `pip installe -e .` from the tensor2tensor directory.
```

Comment:
`pip install -e .` (delete extra `e`)
Also this instruction should follow the `--t2t_usr_dir` explanation (or link to https://github.com/tensorflow/tensor2tensor#adding-your-own-components) because in most cases usr_dir is enough and there is no need for local checkout.

Reply:
Nice; I've clarified that `--t2t_usr_dir` works for testing on a separate project, and `pip install -e .` is only for contributing directly to the tensor2tensor project, along with adding it to all_problems.py.